cc/model/loader.cc

```diff
@@ -0,0 +1,247 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cc/model/loader.h"
```

Comment:
any tests for this?  Roundtrip tests from python?

Reply:
No unit tests just yet. I did verify that the code at HEAD and in this PR both play the same game, and that wrapping and unwrapping a model doesn't change its bytes.